src/pages/404/index.tsx

```diff
@@ -36,23 +53,36 @@ const get404Image = (isMobile: boolean) => {
 export default ({ errorMessage, errorDescription }: { errorMessage?: string; errorDescription?: string }) => {
   const isMobile = useIsMobile()
   const [t] = useTranslation()
+  const history = useHistory()
+
   const isProduction = process.env.NODE_ENV === 'production'
 
   return (
     <Content>
       <NotFoundPanel className="container">
-        <NotFoundImage src={get404Image(isMobile)} alt="404" />
-
-        {(errorMessage || errorDescription) && (
+        {errorMessage || errorDescription ? (
           <>
+            <NotErrorImage src={isMobile ? MobileErrorImage : PCErrorImage} alt="error" />
             <div className={styles.pageCrashedTip}>{t('error.page_crashed_tip')}</div>
-            {!isProduction && (
-              <pre className={styles.pageCrashedError}>
-                {errorMessage}
-                {errorDescription}
-              </pre>
-            )}
+            <button
+              type="button"
+              className={styles.backHomeButton}
+              onClick={() => {
+                history.push('/')
+              }}
+            >
+              BACK HOME
+            </button>
           </>
+        ) : (
+          <NotFoundImage src={get404Image(isMobile)} alt="404" />
+        )}
+
+        {!isProduction && (
+          <pre className={styles.pageCrashedError}>
+            {errorMessage}
+            {errorDescription}
+          </pre>
```

Comment:
I would prefer using a ternary to return `null` explicitly for readability and reduction of potential error